README.md

```diff
@@ -1,4 +1,47 @@
-# math-magicians
-"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:  
-- Make simple calculations. 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magicians
+
+"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) built with React that allows users to:
+- Make simple calculations.
 - Read a random math-related quote.
+## Built With
+
+- HTML&CSS
+- Javascript
+- React
+
+## Getting Started
+To get a local copy up and running follow these simple steps.
+### Prerequisites
+- Having a modern internet browser: I.E. Chrome, Edge, Safari, Firefox.
+### Setup
+- Go into the main page for this project: https://github.com/lu-jim/math-magicians
+- Click the green Code button next to the About section.
+- Select the then click the 📋 button.
+### Install
+- Open your console and change the current working directory into the destination for this project.
+- Type `git clone`, and then paste the URL copied in the Setup section.
+- Type Enter to finish the installation process.
+## Authors
+
+👤 **Luis Fernando Jimenez**
+
+- GitHub: [@lu-jim](https://github.com/lu-jim)
+- Twitter: [@lujimhe](https://twitter.com/lujimhe)
+- LinkedIn: [@lujim](https://www.linkedin.com/in/lujim/)
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel
+## Show your support
+
+Give a ⭐️ if you like this project!
+## Acknowledgments
+
+- Microverse for the guidelines used for this project
+- My SO and project manager, Mane
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL] The MIT link is broken because the MIT file is missing. It would be better to add one. 😉